files/ko/web/css/place-self/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: place-self
+slug: Web/CSS/place-self
+l10n:
+  sourceCommit: c72b86b3d6818ec6c8df1d52a77513d769f4164e
+---
+
+{{CSSRef}}
+
+**`place-self`** [CSS](/ko/docs/Web/CSS) [단축 속성](/ko/docs/Web/CSS/Shorthand_properties) 은 [그리드](/ko/docs/Web/CSS/CSS_grid_layout) 혹은 [플렉스박스](/ko/docs/Web/CSS/CSS_flexible_box_layout) 와 같은 관계형 레이아웃 시스템에서 블록 방향과 인라인 방향에서 (예를 들어, {{CSSxRef("align-items")}} 와 {{CSSxRef("justify-items")}} 속성) 한번에 개별 요소를 정렬합니다. 만일 두번째 값이 지정되지 않는다면, 첫번째 값이 두번째 값으로도 사용됩니다.
+
+{{EmbedInteractiveExample("pages/css/place-self.html")}}
+
+## 구성 속성
+
+이 속성은 다음 CSS의 단축형입니다.
+
+- [`align-self`](/ko/docs/Web/CSS/align-self)
+- [`justify-self`](/ko/docs/Web/CSS/justify-self)
+
+## 구문
+
+```css
+/* 키워드 값 */
+place-self: auto center;
+place-self: normal start;
+
+/* 위치 정렬 */
+place-self: center normal;
+place-self: start auto;
+place-self: end normal;
+place-self: self-start auto;
+place-self: self-end normal;
+place-self: flex-start auto;
+place-self: flex-end normal;
+
+/* 기준선 정렬 */
+place-self: baseline normal;
+place-self: first baseline auto;
+place-self: last baseline normal;
+place-self: stretch auto;
+
+/* 전역 값 */
+place-self: inherit;
+place-self: initial;
+place-self: revert;
+place-self: revert-layer;
+place-self: unset;
+```
+
+### 값
+
+- `auto`
+  - : 부모 속성의 {{cssxref("align-items")}} 값으로 계산됩니다.
+- `normal`
+
+  - : 이 키워드의 효과는 우리가 만든 레이아웃 모드에 따라 달라집니다.
+
+    - 절대 위치로 지정된 레이아웃의 경우, 이 키워드는 _대체된_ 절대 위치 박스에서 `start` 처럼 동작하고, _다른 모든_ 절대 위치 박스에서는 `stretch` 처럼 동작합니다.
+    - 플렉스 요소들에서, 이 키워드는 `stretch` 처럼 동작합니다.
+    - 그리드 요소들에서, 이 키워드는 `stretch` 와 비슷한 동작을 하지만, {{glossary("aspect ratio")}} 박스 형태 혹은 고유한 크기를 가진 박스에서는 `start` 처럼 동작합니다.
+    - 이 속성은 블록 레벨의 박스나 표 셀에 적용되지 않습니다.
+
+- `self-start`
+  - : 요소를 교차 축의 시작 지점에 해당하는 정렬 컨테이너의 가장자리에 맞게 정렬합니다.
+- `self-end`
+  - : 요소를 교차 축의 끝 지점에 해당하는 정렬 컨테이너의 가장자리에 맞게 정렬합니다.
+- `flex-start`
+  - : 플렉스 요소의 교차 시작 마진 가장자리가 선의 교차 시작 가장자리에 맞닿습니다.
+- `flex-end`
+  - : 플렉스 요소의 교차 끝 마진 가장자리가 선의 교차 끝 가장자리에 맞닿습니다.
+- `center`
+  - : 플렉스 요소의 마진 박스가 교차 축의 선 중앙에 위치합니다. 만일 요소의 교차 크기가 플렉스 컨테이너보다 큰 경우, 요소는 양쪽 방향으로 균등하게 넘칩니다.
+- `baseline`, `first baseline`. `last baseline`
+
+  - : 첫번째 혹은 마지막 기준선 정렬에 참여를 지정합니다. 박스의 첫번째 혹은 마지막 기준선 세트의 정렬 기준선을 해당 기준선을 공유하는 그룹의 모든 박스의 공유 첫번째 혹은 마지막 기준선 세트에 있는 기준선과 맞추어 정렬합니다.
+    `first baseline` 의 대체 정렬은 `start` 이며, `last baseline` 의 대체 정렬은 `end` 입니다.
+
+- `stretch`
+  - : 교차 축을 따라 결합된 요소들의 크기가 정렬 컨테이너보다 작은 경우, `auto` 크기의 요소는 크기를 동등하게 (비례적이지 않게) 증가시키는 반면, {{CSSxRef("max-height")}}/{{CSSxRef("max-width")}} (혹은 동등한 기능)에 의해 발생하는 제약들을 존중하며 모든 `auto` 크기의 요소들의 결합된 크기가 교차 축을 따라 정렬 컨테이너를 알맞게 채우도록 합니다.
```

Comment:
```suggestion
  - : 교차 축을 따라 결합된 요소들의 크기가 정렬 컨테이너보다 작고 크기가 `auto`로 지정된 경우, 해당  요소의 크기는 동일하게 (비례하지 않게) 증가하는 반면, {{CSSxRef("max-height")}}/{{CSSxRef("max-width")}} (혹은 동등한 기능)에 의해 발생하는 제약들을 준수하여 모든 `auto` 크기의 요소들의 결합된 크기가 교차 축을 따라 정렬 컨테이너를 정확히 채웁니다.
```

가정문 내 조건 표현을 원문에 맞게 수정 및 서술어 표현을 보다 간단하게 변경해보았습니다. 위와 같이 번역하는 것은 어떨까요? 하단에 원문 첨부드립니다.

원문
```
If the combined size of the items along the cross axis is less than the size of the alignment container and the item is `auto`-sized, its size is increased equally (not proportionally), while still respecting the constraints imposed by {{cssxref("max-height")}}/{{cssxref("max-width")}} (or equivalent functionality), so that the combined size of all `auto`-sized items exactly fills the alignment container along the cross axis.
```